90zfsbootmenu/zfsbootmenu-lib.sh

```diff
@@ -88,14 +88,49 @@ draw_snapshots() {
   benv="${1}"
 
   selected="$( zfs list -t snapshot -H -o name "${benv}" |
-    fzf --prompt "Snapshot > " --tac --expect=alt-x,alt-c \
-        --header="[ENTER] duplicate [ALT+X] clone and promote [ALT+C] clone only [ESC] back" )"
+    fzf --prompt "Snapshot > " --tac --expect=alt-x,alt-c,alt-d \
+        --header="[ENTER] duplicate [ALT+X] clone and promote [ALT+C] clone only [ALT-D] show diff [ESC] back" )"
```

Comment:
Header should be `ALT+D` instead of `ALT-D` for consistency.

Reply:
Good catch, fixed!